spanner/src/update_backup.php

```diff
@@ -25,6 +25,7 @@
 
 // [START spanner_update_backup]
 use Google\Cloud\Spanner\SpannerClient;
+use \DateTime;
```

Comment:
leading slash is redundant for `use` statements
```suggestion
use DateTime;
```